src/com/ConnectionInfoPublisher.hpp

```diff
@@ -0,0 +1,115 @@
+#include <string>
```

Comment:
The alerts from lgtm are not useless:

```suggestion
#pragma once

#include <string>
```